src/commands/create.ts

```diff
@@ -69,11 +70,10 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     const check = /CREATE TABLE/gim.exec(schema.toString());
     if (check) {
       statement = schema;
-    } else if (prefix === undefined) {
-      logger.error(
-        "Must specify --prefix if you do not provide a full Create statement"
-      );
     } else {
+      if (prefix == null || prefix.trim() === "") {
```

Comment:
if `check` is false and prefix doesn't exist, we will create a table without a prefix.